### ISSUE TITLE
Properly Implementing the districtsLocate function

### ISSUE THREAD

**Issue by dammitBrandon** [mcwhittemore/sunlight-congress-api#20]
I have been attempting to use the districtsLocate() function and currently attempting to add a zip code to the call but I am failing miserably at implementing it properly.  I have been looking online and googling for an example but havent been able to find anything and looking through the source code isnt clearing much up for me.  If you could provide a quick example of how to add the zip / lat long to a query before making the call that would be extremely helpful.  Here is an example of what I am trying to do.

```javascript
var district = sunlight.districtsLocate();
  district.search('20744');
//district.filter('zip', '20744'); second failed attempt
  district.call().then(function(data) {
    console.log(data);
  });
```

Thanks in advance Matt!!!

**Comment by dammitBrandon** [comment]
Hey Matthew,

Following up on a PR  I put out for you to check out, it takes care of
adding the zip code or coords to find a district or legislator, I am being
pretty terse right now and I apologize but hopefully the PR makes sense.

Brandon

**Comment by mcwhittemore** [comment]
Closed by #21.

This is now in NPM and tagged at 0.3.1 https://github.com/mcwhittemore/sunlight-congress-api/releases/tag/v0.3.1
